server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/ProjectAnalysisItem.java

```diff
@@ -34,56 +32,56 @@ public ProjectAnalysisItem(SelenideElement elt) {
   }
 
   public ProjectAnalysisItem shouldHaveEventWithText(String text) {
-    elt.find(".project-activity-events").shouldHave(text(text));
+    elt.find(".project-activity-events").shouldHave(Condition.text(text));
     return this;
   }
 
   public ProjectAnalysisItem shouldHaveDeleteButton() {
     elt.find(".js-analysis-actions").click();
-    elt.find(".js-delete-analysis").shouldBe(visible);
+    elt.find(".js-delete-analysis").shouldBe(Condition.visible);
     return this;
   }
 
   public ProjectAnalysisItem shouldNotHaveDeleteButton() {
     elt.find(".js-analysis-actions").click();
-    elt.find(".js-delete-analysis").shouldNotBe(visible);
+    elt.find(".js-delete-analysis").shouldNotBe(Condition.visible);
     return this;
   }
 
   public void delete() {
     elt.find(".js-analysis-actions").click();
     elt.find(".js-delete-analysis").click();
 
-    SelenideElement modal = $(".modal");
-    modal.shouldBe(visible);
+    SelenideElement modal = Selenide.$(".modal");
+    modal.shouldBe(Condition.visible);
     modal.find("button[type=\"submit\"]").click();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "button[type=\"submit\"]" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)